### PR TITLE
Update text in the CONTRIBUTING file.

### PR DESCRIPTION
**Pull request by salticus**
Update the "Testing" section for use in a static site. (The current text seems to be cruft from copy pasting. The git log
does not show an entry for "test" at any point in the history of `package.json`.)

**- Summary**

Update the CONTRIBUTING document with instructions for testing and revising content.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,31 +17,32 @@ $ npm install
 $ npm run build
 ```
 
-## Testing
+## Testing documentation
```
Comment (kingthorin) [inline CONTRIBUTING.md:20]:
```suggestion
## Testing Documentation
```

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Comment by kingthorin** [comment]
@salticus do you plan to finish this?

**Comment by kingthorin** [comment]
@salticus do you plan to finish this?

**Comment by kingthorin** [comment]
This seems to have been abandon by the original contributor, will re-implement.

**Comment by salticus** [comment]
Just noticed this today. Apologies for the abandon.

**Pull request closed without merging**